docs/reference/manifest/extendedpermission.md

```diff
@@ -0,0 +1,55 @@
+---
+title: ExtendedPermission element in the manifest file
+description: ''
+ms.date: 03/04/2020
+localization_priority: Normal
+---
+
+# `ExtendedPermission` element
+
+Defines an extended permission the add-in needs to work. The `ExtendedPermission` element is a child element of [ExtendedPermissions](extendedpermissions.md).
+
+> [!IMPORTANT]
+> This element is only available in the [Outlook add-ins preview requirement set](../objectmodel/preview-requirement-set/outlook-requirement-set-preview.md) against Exchange Online. Add-ins that use this element cannot be published to AppSource or deployed via centralized deployment.
+
+The following are the available values.
+
+|Available value|Description|Add-in types|
+|---|---|---|
+|`AppendOnSend`|Declares that the add-in is using the append on send feature.|`Mail`|
```

Comment:
API instead of feature?

Reply:
and does Mail mean it's available only on mails? If so, this is now also available on Appointments

Reply:
The `Mail` add-in type represents Outlook add-ins.

Reply:
Updated to list hosts instead of add-in type